### DISCUSSION TITLE
Improving Retrospective Demo Engagement

### DISCUSSION THREAD

**Discussion by garethaledavies** [ParabolInc/parabol#5956] [Ideas]
Data from Amplitude shows that only c.3% of folks starting our Retrospective demo get to the group stage, which means the vast majority of people are clicking onto the demo and then almost immediately clicking away.

There's an opportunity to improve demo completion. 

Hypotheses why people abandon the demo:

**1. People want to see how it works before trying it themselves
2. People are impatient and don't want to wait for the bots to write up their reflections**

Some thoughts on each of these points:

> **1. People want to see how it works before trying it themselves**

When I go to a new site to evaluate a technology offering my usual flow is as follows:

- Look for a short video that explains value proposition and shows me what it does 
- If I am convinced: Try the demo next
- If I am not yet convinced: Read the website then either leave (still not convinced), or try the demo (on the road to being convinced).

<img width="1303" alt="Screenshot 2021-01-19 at 15 47 26" src="https://user-images.githubusercontent.com/15236825/105050078-addfea00-5a6d-11eb-82e1-468e21e8235a.png">

We might improve demo engagement if we do more to ease people in with a product video that shows quickly what it does instead of requiring browsers to make the effort of actively engaging with a demo. 

Personal reflection: I remember the first time I came across Parabol's demo, I didn't feel comfortable yet writing in the reflection boxes because even though it was a demo, I didn't want to do it wrong. Doing the demo is a bit like coming into a new team and not really having the psychological safety associated with knowing what's expected.

> **2. People don't want to wait for the bots to write up their reflections**

It takes roughly 35 seconds for all the bots to write up their reflections and we know from user research and observing new folks on our team that the 'Next' button isn't always obvious to people the first time around. 

We might improve this by creating a stronger CTA within the demo nudging people to move on to grouping. It might be that people don't know what to do next or don't realise there is another stage and therefore exit the demo. 

My hunch is that people want to passively see how something works first. If we are engaging people in the demo after they have seen how it works passively, they will have gone to the demo because they are interested (i.e. want to actively participate in the demo) and will feel more confident actively participating because they've seen how it works.

**Comment by garethaledavies** [comment]
In our demo the bot groups across columns – this might be confusing for people encountering the tool for the first time. 

One person commented:

> I'm confused after watching the demo... :grimacing: Why is _Having debates that go nowhere over in group chat_ in 'Continue' ?

**Comment by jordanh** [comment]
This is an underrated discussion! It's going to be difficult for us to resolve this discussion async in GH I feel. If our priorities for the next couple of quarters carry you to action on this one, I think it'd be a good idea to see if there are some folks who'd want to jam on ways if improving the demo experience. We've done _very_ little revision here since we launched it

[a comment by an automated account is not shown]
